projects/observability/src/shared/graphql/request/builders/specification/specification-builder.ts

```diff
@@ -50,4 +74,26 @@ export class SpecificationBuilder {
       })
     };
   }
+
+  private buildSpecName(specs: Specification[]): string {
+    return specs.map(specification => specification.resultAlias()).join('_');
+  }
+
+  private extractForSpecificationsObject(
+    serverData: Dictionary<unknown>,
+    specifications: Dictionary<Specification>
+  ): Dictionary<unknown> {
+    const extractedData: Dictionary<unknown> = {};
```

Comment:
not required since you are doing reduce. can start with `{}` in L96

Reply:
Okay, Thanks